lib/core/test/integration/Test/Integration/Scenario/CLI/Wallets.hs

```diff
@@ -48,13 +49,13 @@ spec = do
     describe "WALLETS_CREATE_05 - Can create wallet with different mnemonic sizes" $ do
         forM_ ["15", "18", "21", "24"] $ \(size) -> it size $ \_ -> do
             let walletName = "Wallet created via CLI"
-            Stdout mnemonics <- generateMnemonicsViaCLI ["--size", size]
-            let passphrase = "Secure passphrase"
-
-            (c, out, err) <- createWalletViaCLI [walletName] mnemonics "\n" passphrase
-            c `shouldBe` ExitSuccess
-            T.unpack err `shouldContain` "Ok.\n"
-            out `shouldContain` walletName
+            checkSimpleCreateWallet
+                (Just "1337")
```

Comment:
What is the relevance of the number `1337`? Is it an arbitrarily-chosen number, or should it be the same as a number somewhere else?